main.js

```diff
@@ -0,0 +1,160 @@
+    }
+    if (
+      cell_2.textContent === "X" && // create if function for each possible "win" statement.
+      cell_5.textContent === "X" &&
+      cell_8.textContent === "X"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_6.textContent === "X" && // create if function for each possible "win" statement.
+      cell_7.textContent === "X" &&
+      cell_8.textContent === "X"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_0.textContent === "O" && // create if function for each possible "win" statement.
+      cell_1.textContent === "O" &&
+      cell_2.textContent === "O"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_0.textContent === "O" && // create if function for each possible "win" statement.
+      cell_3.textContent === "O" &&
+      cell_6.textContent === "O"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_0.textContent === "O" && // create if function for each possible "win" statement.
+      cell_4.textContent === "O" &&
+      cell_8.textContent === "O"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_1.textContent === "O" && // create if function for each possible "win" statement.
+      cell_4.textContent === "O" &&
+      cell_7.textContent === "O"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_3.textContent === "O" && // create if function for each possible "win" statement.
+      cell_4.textContent === "O" &&
+      cell_5.textContent === "O"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_2.textContent === "O" && // create if function for each possible "win" statement.
+      cell_4.textContent === "O" &&
+      cell_6.textContent === "O"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_2.textContent === "O" && // create if function for each possible "win" statement.
+      cell_5.textContent === "O" &&
+      cell_8.textContent === "O"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+    if (
+      cell_6.textContent === "O" && // create if function for each possible "win" statement.
+      cell_7.textContent === "O" &&
+      cell_8.textContent === "O"
+    ) {
+      alert(">CELLS WITHIN CELLS WITHIN CELLS INTERLINKED \n>YOU WIN");
+    }
+  } else {
+    alert("Sorry, you must choose an unclaimed cell.");
+  }
+  whosTurn.innerHTML = `Player ${nextMove} 's Turn!`;
+}
+
+let startButton = document.getElementById("start");
+startButton.addEventListener("click", setupBoard);
+
+function setupBoard(e) {
+  e.target.disabled = true;
```

Comment:
You could also clear the board here (reset all cell textContent to `""`) then if you enable the start button on a win or tie you could easily play another game.